bin/www

```diff
@@ -1,75 +1,72 @@
 #!/usr/bin/env node
 'use strict';
+/* eslint-disable no-console */
 const http = require('http');
 const config = require('config');
 const debug = require('debug')('tnlcf:www');
 
 const app = require('../server');
-const models = require('../db/models');
+const { sequelize } = require('../db/models');
 
 debug(`App environment: ${config.util.getEnv('NODE_ENV')}`);
 
 /**
  * Get port from environment and store in Express.
  */
-var port = normalizePort(process.env.PORT || '3000');
+const port = normalizePort(process.env.PORT || '3000');
 app.set('port', port);
 
 /**
  * Create HTTP server.
  */
-var server = http.createServer(app);
+const server = http.createServer(app);
 
-var startServer = function() {
-    /**
-     * Listen on provided port, on all network interfaces.
-     */
+/**
+ * Listen on provided port, on all network interfaces.
+ */
+function startServer() {
     server.listen(port);
     server.on('error', onError);
     server.on('listening', onListening);
-};
+}
 
-// sync database models
-try {
-    models.sequelize.sync().then(function() {
+/**
+ * Sync database models and start application
+ */
+sequelize
```

Comment:
I noticed whilst working on this that the try/catch syntax here was incorrect (this is async) so needed to be a `.catch` otherwise database errors were never logged.